systemtests/aci_test.go

```diff
@@ -15,39 +15,47 @@ func (s *systemtestSuite) TestACIMode(c *C) {
 	c.Assert(s.cli.GlobalPost(&client.Global{
 		Name:             "global",
 		NetworkInfraType: "aci",
-		Vlans:            "1-4094",
+		Vlans:            "1140-1150",
```

Comment:
Why is vlan range significant for this test?
Tests become less portable if we hard code plan ranges based on a specific setup..